src/main/resources/shoe_customers.avro

```diff
@@ -1,3663 +1,3663 @@
-            {
-                "id": "6143a99d-cd87-473b-bdb3-00ef2230a57b",
-                "first_name": "Ursulina",
-                "last_name": "Bowstead",
-                "email": "ubowstead89@indiegogo.com",
-                "phone": "410-244-2787",
-                "street_address": "04 Fremont Drive",
-                "state": "New York",
-                "zip_code": "11247",
-                "country": "United States",
-                "country_code": "US"
-            },
-            {
-                "id": "2dd93057-0cd6-410f-9f0e-bed259157348",
-                "first_name": "Agnesse",
-                "last_name": "Hannis",
-                "email": "ahannis8a@europa.eu",
-                "phone": "644-556-5359",
-                "street_address": "9 Graceland Circle",
-                "state": "North Carolina",
-                "zip_code": "28247",
-                "country": "United States",
-                "country_code": "US"
-            },
-            {
-                "id": "05e8b91a-8a5a-43a8-a303-ddce87b57868",
-                "first_name": "Perceval",
-                "last_name": "Kix",
-                "email": "pkix8b@feedburner.com",
-                "phone": "213-840-6813",
-                "street_address": "07 Namekagon Court",
-                "state": "Missouri",
-                "zip_code": "64125",
-                "country": "United States",
-                "country_code": "US"
-            },
-            {
-                "id": "96cf6a58-b771-4d5a-875d-80a7430266ac",
-                "first_name": "Gilda",
-                "last_name": "Pagnin",
-                "email": "gpagnin8c@umich.edu",
-                "phone": "930-175-6797",
-                "street_address": "5098 Grayhawk Pass",
-                "state": "Illinois",
-                "zip_code": "60630",
-                "country": "United States",
-                "country_code": "US"
-            }
-        ]
+  "namespace": "shoes",
+  "name": "shoe_customers",
+  "type": "record",
+  "fields": [
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name": "first_name",
+      "type": "string"
+    },
+    {
+      "name": "last_name",
+      "type": "string"
+    },
+    {
+      "name": "email",
+      "type": "string"
+    },
+    {
+      "name": "phone",
+      "type": "string"
+    },
+    {
+      "name": "street_address",
+      "type": "string"
+    },
+    {
+      "name": "state",
+      "type": "string"
+    },
+    {
+      "name": "zip_code",
+      "type": "string"
+    },
+    {
+      "name": "country",
+      "type": "string"
+    },
+    {
+      "name": "country_code",
+      "type": "string"
     }
+  ],
+  "arg.properties": {
+    "options": [
+      {
+        "id": "01a429f8-f576-4d9b-8d0e-4d5e8ef37e09",
+        "first_name": "Cyrill",
+        "last_name": "Haeslier",
+        "email": "vbiaggelli0@prweb.com",
+        "phone": "615-978-0286",
+        "street_address": "7 Hooker Court",
```

Comment:
Do these all have the same email, phone, street address? Do you have a script generating these and would it be easy to vary them?

Reply:
Oops, no this is a bug in my script. I'll update it. I've taken the data from mockeroo

Reply:
I am not using regex to generate these since that would make data kind of inconsistent, same id but different emails etc.

Reply:
thanks!